google/cloud/aiplatform/metadata/metadata.py

```diff
@@ -30,28 +30,49 @@ class _MetadataService:
     def __init__(self):
         self._experiment = None
         self._run = None
+        self._metric = None
+        self._experiment_resource = None
```

Comment:
probably just repurpose self._experiment to actually store the resource? same with others...

Reply:
done.